_posts/2019-07-12-service-design-anti-patterns.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Service Design Anti-patterns
+author: sam_quayle
+---
+
+![A platform at a tube station which says 'Mind the gap'!]({{ "/images/mind_the_gap.jpg"}})
+
+Despite its recent heightened profile – thanks in no small part to the efforts of the UK’s Government Digital Services – the field of Service Design is not as new as you might think. According to [this article](https://service-design.co/book-a-tiny-history-of-service-design-368ed603797c), you can trace its origins back to the turn of the 20th century with Taylorism and Human Factors Engineering through to the 1980s when Lynn Shostack first coined the term ‘Service Design’ and it began to emerge as a practice in its own right. Since then, there has been no shortage of articles, books, talks and consultancies describing the benefits, best practices and success stories of this burgeoning field. But what about the failures? What are the pitfalls, the misconceptions, the anti-patterns that can scupper even the most well-intentioned teams?
+
+Here are the most common mistakes I’ve encountered (and made), over the last five years delivering services, and how to avoid them:
+
+# Unempowered Service Managers
+Turning service designing thinking from a plan into implementation takes a lot of work. Often, it requires orchestrating change across a range of areas within an organisation: aligning teams, processes and communications. With this in mind, the person responsible for delivering a service needs to have the knowledge, relationships and authority to make the big decisions required to catalyse the required change. A Product Owner, alone, won’t cut it. Product Owners should be left to do what they do best: Make timely decisions, set priorities and represent the users throughout the delivery process. An experienced, well-connected Service Manager can not only provide support and escalation for Products Owners but can often be the difference between delivering a cohesive service and disjointed one. Or worse, not delivering a service at all. 
+
+# Someone’s needs, first 
+Whilst early incarnations of Service Design were focussed on improving operational processes, starting your design process from the needs of ‘the business’ can often lead teams to lose sight of who they’re providing a service for and why. Inversely, ignoring the needs of the business can lead to impractical solutions and can also have a negative effect on the end user's experience. For example, reducing the number of questions in a form to make it quicker and easier to complete will increase completion rates and user satisfaction. However, it might also make it harder for someone to review, leading to longer processing times, additional requests for information and reducing satisfaction on both sides. Taking a balanced view of the service users and providers needs and considering the impact that certain interventions will have is key to actually improving a service, not just shifting the burden. 
```

Comment:
Suggest rewording _the service users and providers needs_ to _the needs of both service users and providers_.